packages/mds-api-server/middleware/prometheus/index.ts

```diff
@@ -0,0 +1,49 @@
+import type { Labels as PrometheusLabels, Opts as PrometheusOptions, TransformLabelsFn } from 'express-prom-bundle'
+import prometheus from 'express-prom-bundle'
+import promClient from 'prom-client'
+import { ProviderIdLabeler } from './provider-id-labeler'
+import { ApiRequest, ApiResponse, ApiResponseLocals, ApiClaims } from '../../@types'
+
+type PrometheusLabeler = {
+  label: string
+  base: string | number | null
+  transformer: TransformLabelsFn
+}
```

Comment:
Is this type not exported from either `express-prom-bundle`? or `prom-client`?

Reply:
Nope, this is a custom type to use with the helpers for composition I wrote